socorro/processor/mozilla_processor_2015.py

```diff
@@ -1,6 +1,8 @@
-from ujson import dumps
+import ujson
```

Comment:
when Rob first introduced ujson into the project, we discovered incompatibilities since it doesn't support ```object_hooks```.  I recall the he and I agreed that we would use ujson where it was compatible and only use the native json when we absolutely had to.